packages/nimble-blazor/NimbleBlazor/Components/NimbleRadioButton.razor

```diff
@@ -0,0 +1,10 @@
+@namespace NimbleBlazor
+@inherits ComponentBase
+<nimble-radio-button value="@Value"
+                     current-value="@CurrentValue"
```

Comment:
I don't believe we need to set both `value` and `current-value`. At the very least, the [FAST documentation](https://github.com/microsoft/fast/blob/218cfc014f5c76464493725f83bd883e8b07defa/packages/web-components/fast-foundation/src/form-associated/form-associated.ts#L338) doesn't lead me to believe we need to do this.

Note that no other Blazor components set both of these properties, and it seems we favor `current-value`.

Reply:
Removed value